
wrapper:
  mode: realtime #offline or realtime
  bag_path: /dataset/nclt/20120115.bag
  save_pcd: false
  speed_factor: 41.0
  config_file_name: nclt.yaml
  lidar_topic: /os_cloud_node/points
  imu_topic: /os_cloud_node/imu/data_raw
  lidar_type: 2 # velodyne
  point_filter_num: 6          
  scan_line: 128
  blind: 4
front_end:
  filter_leaf_size: 0.5
  extrin_r: [1,1,0,
            0,1,0,
            0,0,1]
  # extrin_r: [0.9998477, -0.0174524, 0,
  #          0.0174524,  0.9998477, 0,
  #          0,          0,         1]
  extrin_t: [0,0,0.28]
  trajectory_save: false
  trajectory_save_file: nclt9.txt
ieskf:
  cov_gyroscope: 0.1
  cov_acceleration: 0.1
  cov_bias_acceleration: 0.001
  cov_bias_gyroscope: 0.001

local_map:
  map_side_length_2: 100.0
  map_resolution: 0.5

